### DISCUSSION TITLE
Programmatically open a cell editor

### DISCUSSION THREAD

**Discussion by SreeniIO** [glideapps/glide-data-grid#674] [Q&A]
Is there a way to programmatically open a cell editor? I'm looking for a way to programmatically emit click and double-click events on a cell in both header and body. And a way to programmatically close any open editor.

**Comment by Higgy831** [comment]
This would be really great! Or is there a way to skip the "Enter"/doubleclick events, so whenever a cells gets focused the editor opens?
